### PR TITLE
{bio}[goolf/1.7.20] BamBam v1.4

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['bam2consensus', 'bam2fastq', 'catBam', 'counter',
+			             'eflen', 'gapfall', 'gardener', 'giraf', 'hapHunt',
+                                     'hmmph', 'indelible', 'interSnp', 'metHead', 'pebbles',
+                                     'polyCat', 'polyCat-methyl', 'polyDog', 'subBam']],
+    'dirs': [],
+}
+
+modextrapaths = {
+	'PATH': 'scripts',
```
Comment (boegel) [inline easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb]: @pescobar Don't use tabs please, use 4 spaces instead.

File: easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+sources = ['%(namelower)s-%(version)s.tgz']
+checksums = ['a9b178251d771aafb8c676d30a9af88ea2fc679c2a5672b515f86be0e69238f1']
+
+unpack_options = '--strip-components=1'
+
+dependencies = [
+    ('Perl', '5.22.2'),
+    ('BamTools', '2.4.0'),
+    ('HTSlib', '1.3.1'),
+    ('SAMtools', '1.3.1'),
```
Comment (boegel) [inline easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb]: @pescobar Missing easyconfig for this dependency?

File: easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+]
+
+skipsteps = ['configure', 'install']
+
+buildininstalldir = True
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['bam2consensus', 'bam2fastq', 'catBam', 'counter',
+			             'eflen', 'gapfall', 'gardener', 'giraf', 'hapHunt',
```
Comment (boegel) [inline easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb]: indent is off by one space here...

File: easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+dependencies = [
+    ('Perl', '5.22.2'),
+    ('BamTools', '2.4.0'),
+    ('HTSlib', '1.3.1'),
+    ('SAMtools', '1.3.1'),
+]
+
+skipsteps = ['configure', 'install']
+
+buildininstalldir = True
```
Comment (boegel) [inline easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb]: @pescobar Hold on, isn't `MakeCp` better/cleaner here?

Comment (pescobar) [reply easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb]: no strong preference. As you wish

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2106.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/10237b448993a2e7dc7c936f7b2401f3 for a full test report.

File: easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb (hunk 4 above)
Comment (boegel) [reply easybuild/easyconfigs/b/BamBam/BamBam-1.4-goolf-1.7.20.eb]: Actually, won't make much of a difference, there's more stuff there than just the binaries...

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2106.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b7c42b6586426a6ba932a93f360ce4a3 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**